packages/vuetify/src/components/VCalendar/VCalendar.ts

```diff
@@ -98,7 +98,7 @@ export default CalendarWithEvents.extend({
           break
         case '4day':
           component = VCalendarDaily
-          end = relativeDays(copyTimestamp(end), nextDay, 4)
+          end = relativeDays(copyTimestamp(end), nextDay, 3)
```

Comment:
`relativeDays` working with `nextDay` is supposed to calculate end date from the next date of start date. e.g. If start date is `1` and days span is `1`, it will return 2. Therefore, when using `type` as `4day` it includes the start date, so days span shall be set as `3` here